cpp/daal/include/algorithms/decision_forest/decision_forest_classification_model_builder.h

```diff
@@ -110,22 +110,14 @@ class DAAL_EXPORT ModelBuilder
     *  \param[in] cover           Cover (Hessian sum) of the node
     *  \return Node identifier
     */
-    NodeId addLeafNode(const TreeId treeId, const NodeId parentId, const size_t position, const size_t classLabel, const double cover)
+    NodeId addLeafNode(const TreeId treeId, const NodeId parentId, const size_t position, const size_t classLabel, const double cover = 0.0)
     {
         NodeId resId;
         _status |= addLeafNodeInternal(treeId, parentId, position, classLabel, cover, resId);
         services::throwIfPossible(_status);
         return resId;
     }
 
-    /**
-    *  \DAAL_DEPRECATED
-    */
-    DAAL_DEPRECATED NodeId addLeafNode(const TreeId treeId, const NodeId parentId, const size_t position, const size_t classLabel)
-    {
-        return addLeafNode(treeId, parentId, position, classLabel, 0);
-    }
```

Comment:
Major binary version of library should be increased since ABI would change and I'm unsure if it's feasible for next major release as some additional work related to versioned shared libraries is needed.
I'm OK with other non-deletion changes, but you should verify binary backward compatibility on both of Linux and Windows.

Reply:
Do you suggest not deleting for now?

Reply:
Yes, postpone PR merge until next major release and then bump major binary version.